packages/zoe/src/contracts/coveredCall.js

```diff
@@ -92,7 +92,12 @@ const start = zcf => {
 
   const creatorInvitation = zcf.makeInvitation(makeOption, 'makeCallOption');
 
-  return harden({ creatorInvitation });
+  const creatorFacet = harden({
+    getDonePromise: () => zcf.getDonePromise(),
+    getAdminData: () => zcf.getAdminData(),
+  });
+
+  return harden({ creatorInvitation, creatorFacet });
```

Comment:
Hmm, I don't think we should include this. CoveredCall is one of our most public contracts and it's important that it's as minimal as possible. I don't think this information is needed for the use case in particular. Can we choose a less-used, more example than core functionality contract?